assets/js/index.js

```diff
@@ -0,0 +1,44 @@
+import { webNnStatus } from './common_utils.js'
+
+async function webNNChecker() {
+  const webnnStatus = await webNnStatus();
```

Comment:
(nit)
`webnnStatus`
`webNnStatus`

Reply:
Thanks for the feedback, fixed in new commit https://github.com/microsoft/webnn-developer-preview/pull/43/commits/9636b15aae9e0e02096b2c3e78f8bcbdc8ffb1f2 , PTAL